backend/tests/test_sessions_routes.py

```diff
@@ -274,6 +277,34 @@ def test_no_readonly_session_as_user(client, db, user, kubernetes):
     assert not kubernetes.sessions
 
 
+def test_create_readonly_session_as_user(client, db, user, kubernetes):
```

Comment:
```suggestion
def test_create_multiple_readonly_sessions_as_user(client, db, user, kubernetes):
```

I'd choose a more describing name here.